backend/app/routes/contactUs/delete.js

```diff
@@ -1,55 +1,26 @@
-const to = require('await-to-js').default;
-const { ErrorHandler } = require('../../../helpers/error');
-const constants = require('../../../constants');
-const Contact = require("../../models/contactUs");
-const Admin = require("../../models/Admin");
+const mongoose = require('mongoose');
+const ContactUs = require('../../models/contactUs');
 
-// Controller to delete a contact by adminid and contactdocumentid
 module.exports = async (req, res, next) => {
-    const { contactId, adminId } = req.body;
+  try {
+    const payload = res.locals.decode;
+    const { contactUsId } = req.body;
 
-    // Check if contactId is provided
-    if (!contactId || !adminId) {
-        const error = new ErrorHandler(constants.ERRORS.VALIDATION, {
-            statusCode: 400,
-            message: 'Validation Error',
-            errStack: 'Both IDs are required to delete a contact',
-        });
-        return next(error);
+    if (!payload.isSuperAdmin) {
```

Comment:
[ques] why are we doing this check? shouldn't auth middleware take care of it?

Reply:
@Kajol-Kumari in every file they is this type of check. So I thought Auth Middleware doesn't handle it.

Reply:
can u provide me the reference of where do u see it, it shouldn't be required ideally, try removing and then test the delete functionality, it should not work without authorization key of an admin

Reply:
can you please check this here i saw like this [Here](https://github.com/HITK-TECH-Community/Community-Website/blob/main/backend/app/routes/admin/deleteAdmin.js)

Reply:
@Kajol-Kumari  middleware is not handdling it. it passing the error or verified data to the next. we have to check. It is needed. you can check the middleware [Here](https://github.com/HITK-TECH-Community/Community-Website/blob/main/backend/helpers/middlewares/auth.js) It throws error only if token is missing.